src/main/java/de/tum/in/www1/artemis/web/websocket/dto/SubmissionPatchPayload.java

```diff
@@ -0,0 +1,7 @@
+package de.tum.in.www1.artemis.web.websocket.dto;
+
+import de.tum.in.www1.artemis.domain.SubmissionPatch;
+import de.tum.in.www1.artemis.domain.User;
+
+public record SubmissionPatchPayload(SubmissionPatch submissionPatch, User sender) {
```

Comment:
Do you really need the full user including all its details? I don't think so, please limit the data sent over the network according to the guidelines mentioned on https://docs.artemis.cit.tum.de/dev/guidelines/server.html#use-dtos-for-data-transfer